### PR TITLE
feat(AI Transform Node): Support for drag and drop

### PR DESCRIPTION
**Pull request by michael-radency**
## Summary

Allow dragging data pills from the input to insert the field name

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/NODE-1863/input-fields-dragging

### CODE REVIEW

File: packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue [hunk 1]
```diff
@@ -277,4 +326,11 @@ onMounted(() => {
 	color: var(--color-warning);
 	line-height: 1.2;
 }
+.droppable {
+	border: 1.5px dashed var(--color-ndv-droppable-parameter) !important;
+}
+.activeDrop {
+	border: 1.5px solid var(--color-success) !important;
```
Comment (ShireenMissi) [inline packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue:333]: can this be done without !important ?

File: packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue [hunk 2]
```diff
@@ -227,7 +276,7 @@ onMounted(() => {
 
 <style module lang="scss">
 .input * {
-	border: 0 !important;
+	border: 1.5px transparent !important;
```
Comment (ShireenMissi) [inline packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue:279]: can this be done without !important ?

Comment (michael-radency) [reply packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue:279]: no, we need to override defaults of N8nInput(ElInput)

File: packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue (hunk 1 above)
Comment (michael-radency) [reply packages/editor-ui/src/components/ButtonParameter/ButtonParameter.vue:333]: no, we need to override defaults of N8nInput(ElInput)

**Review by ShireenMissi — APPROVED** [review]
LGTM 🎉

**Comment by janober** [comment]
Got released with [`n8n@1.70.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.70.0)

**Pull request merged**